src/java/org/apache/cassandra/db/guardrails/GuardrailsMBean.java

```diff
@@ -553,4 +553,21 @@
      */
     void setMinimumReplicationFactorThreshold (int warn, int fail);
 
+    /**
+     * @return The threshold to fail when replication factor is greater threshold.
+     */
+    int getMaximumReplicationFactorWarnThreshold();
+
+    /**
+     * @return The threshold to fail when replication factor is greater threshold.
+     */
+    int getMaximumReplicationFactorFailThreshold();
+
+    /**
+     * @param warn the threshold to warn when the maximum replication factor is greater than
+     *             threshold -1 means disabled.
+     * @param fail the threshold to fail when the maximum replication factor is greater than
+     *             threshold -1 means disabled.
```

Comment:
```suggestion
     * @param warn The threshold to warn when the maximum replication factor is greater than
     *             threshold. -1 means disabled.
     * @param fail The threshold to fail when the maximum replication factor is greater than
     *             threshold. -1 means disabled.
```
This also applies to the equivalent `setMinimumReplicationFactorThreshold`.

Reply:
As mentioned above, I would also apply this fix to `setMinimumReplicationFactorThreshold`.